service/kernel/service_type_traits.h

```diff
@@ -24,6 +24,36 @@ namespace services
 {
 namespace internal
 {
+template <CpuType cpu, typename T>
+struct RemovePointer
+{
+    typedef T type;
+};
+
+template <CpuType cpu, typename T>
+struct RemovePointer<cpu, T *>
+{
+    typedef T type;
+};
+
+template <CpuType cpu, typename T>
+struct RemovePointer<cpu, T * const>
+{
+    typedef T type;
+};
+
+template <CpuType cpu, typename T>
+struct RemovePointer<cpu, T * volatile>
+{
+    typedef T type;
+};
+
+template <CpuType cpu, typename T>
+struct RemovePointer<cpu, T * const volatile>
+{
+    typedef T type;
+};
+
```

Comment:
What is the reason to introduce it?

Reply:
It's a more elegant way to get type of pointer instead of usage smth. like ```RemoveReference<cpu, decltype(*pointer)>```

Reply:
Do you use it somewhere?

Reply:
Removed